v2/thirdpartyemailpassword/migration/bulk-user-import.mdx

```diff
@@ -0,0 +1,253 @@
+---
+id: bulk-user-import
+title: Bulk User Import
+hide_title: true
+---
+
+import CoreInjector from "/src/components/coreInjector";
+import Tabs from '@theme/Tabs';
+import SelfHostedTabs from "/src/components/tabs/SelfHostedTabs"
+import TabItem from '@theme/TabItem';
+
+# Bulk User Import
+
+The Bulk User Import API can be used to import users with both `emailpassword` and `thirdparty` recipes simultaneously. Learn more about this API in the  [CDI specs](https://app.swaggerhub.com/apis/supertokens/CDI/5.1.0#/Bulk%20Import/getBulkImportUsers).
+
+SuperTokens currently supports importing users with `BCrypt`, `Argon2` or `Firebase Scrypt` password hashes.
+
+:::important
+If you do not have access to your user's password hashes, you can use our [guide for migrating them dynamically during login](./ep-migration-without-password-hash).
+:::
+
+## Step 1: Prepare the import JSON file
+
+Convert your existing authentication provider data to the required format as specified [here](./bulk-user-import-schema-examples.mdx). An example JSON would look like this:
+
+```json
+{
+  "users": [
+    {
+      "loginMethods": [
+        {
+          "recipeId": "emailpassword",
+          "tenantIds": ["public"],
```

Comment:
should `tenantIds` with `["public"]` be a mandatory input? this would be applicable only for importing multi-tenant users right?

Reply:
No, it is not a mandatory input. I only included it make it obvious that the users will be imported to the `public` tenant.